codegen/test/api/things.test.io/v1/proto_deepcopy.go

```diff
@@ -31,3 +31,15 @@ func (in *OilType) DeepCopyInto(out *OilType) {
 	p := proto.Clone(in).(*OilType)
 	*out = *p
 }
+
+// DeepCopyInto supports using AttributeManifest within kubernetes types, where deepcopy-gen is used.
+func (in *PaintSpec) DeepCopyInto(out *PaintSpec) {
+	p := proto.Clone(in).(*PaintSpec)
+	*out = *p
+}
+
+// DeepCopyInto supports using AttributeManifest within kubernetes types, where deepcopy-gen is used.
+func (in *OilType) DeepCopyInto(out *OilType) {
+	p := proto.Clone(in).(*OilType)
+	*out = *p
+}
```

Comment:
what produced this change? just curious

Reply:
To be honest I'm not sure, I was wondering the same thing.